components/automate-deployment/Makefile

```diff
@@ -258,34 +258,6 @@ a1-migration-create-minimal-artifact: a1-migration-hab-keys
 a1-migration-upload-artifact: hab-auth-token
 	cd a1-migration && docker-compose exec a1-migration bash -c 'HAB_ORIGIN=devchef HAB_AUTH_TOKEN=${HAB_AUTH_TOKEN} hab pkg upload /a1-migration/results/*.hart'
 
-proxy-host-setup:
```

Comment:
I think this implies you can `git rm -f components/automate-deployment/proxy` as well